src/inc/gcinfoencoder.h

```diff
@@ -27,7 +27,8 @@
                 hasPSPSymStackSlot,
                 hasGenericsInstContextStackSlot, 
                 hasStackBaseregister,
-                wantsReportOnlyLeaf,
+                wantsReportOnlyLeaf (AMD64 JIT64 use only),
```

Comment:
I would say "AMD64 use only", not just JIT64. Yes, if we didn't have to support JIT64 anymore, we could remove it, but per the ABI doc, "(JIT32 and RyuJIT set it, JIT64 doesn't)".